src/cascadia/TerminalCore/terminalrenderdata.cpp

```diff
@@ -174,6 +174,7 @@ void Terminal::SelectNewRegion(const COORD coordStart, const COORD coordEnd)
 
     SetSelectionAnchor(realCoordStart);
     SetEndSelectionPosition(realCoordEnd);
+    _buffer->GetRenderTarget().TriggerSelection();

```

Comment:
What's this do for this CR?

Reply:
Super tiny/unrelated issue. Making a selection would not signal the UiaRenderer in a timely fashion in this case. Can I keep it?

Reply:
lol sure